src/main/java/uk/nhs/digital/emailadapter/schedulingtasks/EmailScheduledTask.java

```diff
@@ -0,0 +1,180 @@
+import microsoft.exchange.webservices.data.search.filter.SearchFilter.SearchFilterCollection;
+import uk.nhs.digital.emailadapter.service.StagedStopwatch;
+import uk.nhs.digital.emailadapter.transformer.PDFTransformer;
+
+@Slf4j
+@EnableAsync
+@Getter
+@Setter
+@Component
+public class EmailScheduledTask {
+
+  private ExchangeService service = new ExchangeService(ExchangeVersion.Exchange2010_SP2);
+  private AWSSimpleSystemsManagement ssm = AWSSimpleSystemsManagementClientBuilder.defaultClient();
+  StandardPBEStringEncryptor decryptor = new StandardPBEStringEncryptor();
+  private String mpCryptoPassword;
+
+  @Autowired
+  public EmailScheduledTask(@Value(value = "${mpCryptoPassword}") final String mpCryptoPassword)
+      throws Exception {
+    this.mpCryptoPassword = mpCryptoPassword;
+    decryptor.setPassword(mpCryptoPassword);
+    ExchangeCredentials credentials = new WebCredentials(
+        decryptor.decrypt(getParameter("username")), decryptor.decrypt(getParameter("password")));
+    service.setCredentials(credentials);
+    service.autodiscoverUrl(decryptor.decrypt(getParameter("username")));
+  }
+
+  @Async
+  @Scheduled(fixedRateString = "${fixedRate.in.milliseconds}",
+      initialDelayString = "${initialDelay.in.milliseconds}")
+  public void sendMails() throws InterruptedException {
+    String timeStamp =
+        DateTimeFormatter.ofPattern("yyyy-MM-dd_HH:mm:ss").format(LocalDateTime.now());
+    log.info("Invocation started: {}", timeStamp);
+
+    try {
+      StagedStopwatch stopwatch = StagedStopwatch.start();
+      ItemView view = new ItemView(10);
+      SearchFilterCollection searchFilterCollection =
+          new SearchFilter.SearchFilterCollection(LogicalOperator.And);
+      searchFilterCollection.add(
+          new SearchFilter.IsEqualTo(EmailMessageSchema.From, getParameter("EMS_REPORT_SENDER")));
+      searchFilterCollection.add(new SearchFilter.IsEqualTo(EmailMessageSchema.IsRead, false));
+      FindItemsResults<Item> findResults =
+          service.findItems(WellKnownFolderName.Inbox, searchFilterCollection, view);
+
+      while (findResults.getTotalCount() > 0) {
+        for (Object item : findResults.getItems()) {
+          EmailMessage emailMessage = (EmailMessage) item;
+          emailMessage
+              .load(new PropertySet(BasePropertySet.FirstClassProperties, ItemSchema.MimeContent));
+          log.info("attachment count: {} ", emailMessage.getAttachments().getCount());
+
+          Attachment attachment = emailMessage.getAttachments().getItems().get(0);
+          stopwatch.finishStage("Getting html attchement from email");
+
+          if (attachment instanceof FileAttachment) {
+            FileAttachment fileAttachment = (FileAttachment) attachment;
+            fileAttachment.load();
+
+            // convert bytes[] to string
+            String htmlString = new String(fileAttachment.getContent(), StandardCharsets.UTF_8);
```

Comment:
check the type of attachment to verify if it is html rather than assume it is html